apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/connect-term-to-video.e2e.cy.ts

```diff
@@ -27,7 +29,9 @@ describe(`term detail flow`, () => {
 
                 cy.getByDataAttribute('term').click();
 
-                cy.get(`[data-testid="${termId}"] > :nth-child(1)`).click();
+                // cy.get(`[data-testid="term/${termId}"] > :nth-child(1)`).click();
+
+                cy.get(`[href="/Resources/Terms/${termId}"]`).click();
```

Comment:
👍

Reply:
Might be nice to make this a Cypress command in the future.